checks/libs/wmi/sampler.py

```diff
@@ -91,8 +92,10 @@ class WMISampler(object):
 
     def __init__(self, logger, class_name, property_names, filters="", host="localhost",
                  namespace="root\\cimv2", provider=None,
-                 username="", password="", and_props=[], timeout_duration=10):
+                 username="", password="", and_props=[],
+                 mute=True, timeout_duration=10):
```

Comment:
Yeah, I had looked at the top of the PR yesterday, and forgot about `mute` being `False` there. Thanks for the explanation though.